data_util.py

```diff
@@ -2,10 +2,15 @@
 import os
 from os import path
 import urllib
+import urllib.request as request
```

Comment:
Python 2 doesn't have urllib.request. This import will make the code not able to run in Python 2.